src/app/loadout-drawer/loadout-types.ts

```diff
@@ -9,15 +8,7 @@ export interface LoadoutItem {
 }
 
 /** In memory loadout structure. */
-export interface Loadout {
-  id: string;
-  classType: DestinyClass;
-  name: string;
+export type Loadout = Omit<DimApiLoadout, 'equipped' | 'unequipped'> & {
```

Comment:
This means we don't have to keep the DIM loadout and API loadout types in sync anymore.